### DISCUSSION TITLE
Welcome to mudaeautoclaimer-roller Discussions!

### DISCUSSION THREAD

**Discussion by lilyyy411** [4gboframram/mudaeautoclaimer-roller#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Ethan-Saddler** [comment]
Quick question: does this work on other people's rolls? Love the bot so far :)
